subsys/bluetooth/shell/gatt.c

```diff
@@ -950,6 +950,70 @@ static int cmd_notify(const struct shell *sh, size_t argc, char *argv[])
 	return 0;
 }
 
+#if defined(CONFIG_BT_GATT_NOTIFY_MULTIPLE)
+static int cmd_notify_mult(const struct shell *sh, size_t argc, char *argv[])
+{
+	const size_t max_cnt = CONFIG_BT_L2CAP_TX_BUF_COUNT;
+	struct bt_gatt_notify_params params[max_cnt];
+	const size_t min_cnt = 1U;
+	unsigned long data;
+	unsigned long cnt;
+	uint16_t cnt_u16;
+	int err = 0;
+
+	if (!echo_enabled) {
+		shell_error(sh, "No clients have enabled notifications for the vnd1_echo CCC.");
+
+		return -ENOEXEC;
+	}
+
+	cnt = shell_strtoul(argv[1], 10, &err);
+	if (err != 0) {
+		shell_error(sh, "Invalid count parameter: %s", argv[1]);
+
+		return -err;
+	}
+
+	if (!IN_RANGE(cnt, min_cnt, max_cnt)) {
+		shell_error(sh, "Invalid count value %lu (range %zu to %zu)",
+			    cnt, min_cnt, max_cnt);
+
+		return -ENOEXEC;
+	}
+
+	cnt_u16 = (uint16_t)cnt;
+
+	if (argc > 2) {
+		data = shell_strtoul(argv[2], 16, &err);
+		if (err != 0) {
+			shell_error(sh, "Invalid data parameter: %s", argv[1]);
+
+			return -err;
+		}
+	}
+
+	(void)memset(&params, 0, sizeof(params));
```

Comment:
`&params` means "pointer to pointer to first element of params". Implicit decay strikes again.
```suggestion
	(void)memset(&params[0], 0, sizeof(params));
```
```suggestion
	(void)memset(params, 0, sizeof(params));
```